packages/webdriver/src/request/browser.ts

```diff
@@ -38,7 +38,7 @@ export default class BrowserRequest extends WebDriverRequest {
         }
 
         if (options.username && options.password) {
-            const encodedAuth = btoa(`${options.username}:${options.password}`)
```

Comment:
I'm not sure is it executed inside Browser or Node environment?

Reply:
always inside Node environment